dictionaries/en_AU/src/additional_words.txt

```diff
@@ -10,3 +10,86 @@
 # There should be one entry per line.
 #
 # cspell-tools: split
+
+# Australian slang and colloquialisms
+arvo
+barbie
+brekkie
+chook
+dunny
+esky
+mozzie
+sanga
+servo
+sickie
+tinnie
+tradie
+ute
+yakka
+bonza
+ripper
+strewth
+crikey
+cobber
+drongo
+galah
+larrikin
+ocker
+swag
+billabong
+boomerang
+corroboree
+didgeridoo
+dreamtime
+
+# Australian food and drink
+lamington
+pavlova
+Anzac
+Vegemite
+Milo
+TimTam
+flatwhite
+babycino
+parma
+schnitty
+
+# Australian occupations (informal)
+ambo
+firey
+postie
+sparkie
+chippie
+brickie
+truckie
+bikie
+surfie
+bushie
+
+# Australian place names (major cities and landmarks)
+Melbourne
+Sydney
+Brisbane
+Perth
+Adelaide
+Hobart
+Darwin
+Canberra
+Queensland
+Tasmania
+Kakadu
+Uluru
+
+# Australian wildlife
+kangaroo
+koala
+wombat
+echidna
+platypus
+dingo
+kookaburra
+wallaby
+bilby
+quokka
+numbat
+bandicoot
```

Comment:
I was surprised to see them added **only** to en_AU

So I opened

- https://github.com/streetsidesoftware/cspell-dicts/pull/4581